### PR TITLE
Fix #1330

### LINKED ISSUE #1330: Point out that errors handling only works in API platform routes
I think that in the documentation we should add a notice that the exception to status feature only works in routes managed by API platform

https://github.com/api-platform/core/issues/3239#issuecomment-809726514

Should I submit a PR?

### PR DESCRIPTION
**Pull request by vincentchalamon**

### CODE REVIEW

File: core/errors.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ or in the format described in the [RFC 7807](https://tools.ietf.org/html/rfc7807
 
 ## Converting PHP Exceptions to HTTP Errors
 
-The framework also allows you configure the HTTP status code sent to the clients when custom exceptions are thrown.
+The framework also allows you to configure the HTTP status code sent to the clients when custom exceptions are thrown
+on an API Platform resource operation.
```
Comment (guilliamxavier) [inline core/errors.md:12]: How about mentioning `_api_respond: true` for custom operations?

Comment (vincentchalamon) [reply core/errors.md:12]: Could work, but sounds tricky to me, like using API Platform to handle Symfony stuffs. WDYT @api-platform/core-team ?

Comment (alanpoulain) [reply core/errors.md:12]: Why talking here about the `_api_respond`?

Comment (vincentchalamon) [reply core/errors.md:12]: Because of https://github.com/api-platform/core/issues/3239#issuecomment-810008374

**Comment by alanpoulain** [comment]
Why not `2.6`?

**Comment by vincentchalamon** [comment]
@alanpoulain It's not a bug in my opinion

**Review by alanpoulain — APPROVED** [review]

**Pull request merged**